server/controllers/auth/login.js

```diff
@@ -0,0 +1,68 @@
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+
+const loginValidation = require('../validation/loginValidation');
+const Admin = require('../../database/models/admin');
+const Client = require('../../database/models/client');
+
+require('env2')('config.env');
+
+const login = async (req, res) => {
+  const { email, password, mobileNumber } = req.body;
+  try {
+    const valid = await loginValidation(email, mobileNumber, password);
+    if (!valid) {
+      res.status(400).json({
+        status: 'failed',
+        message: 'Check the entered data',
+      });
+    } else {
+      const clients = await Client.findOne({
+        $or: [{ email }, { mobileNumber }],
+      });
+      await bcrypt.compare(password, clients.password, (err, result) => {
+        if (!result)
+          res.status(400).json({
+            status: 'failed',
+            message: 'somthing wrong with Email or passoword',
+          });
+        const userToken = { userId: clients.id };
+        const cookie = sign(userToken, process.env.SECRET_KEY);
+        res.cookie('client', cookie).json({
+          status: 'successfully',
+          role: 'applicant',
+          data: {
+            email,
+            mobileNumber,
+            fullName: `${clients.fullName}`,
+            avatar: `${clients.avatar}`,
```

Comment:
I don't know if its best practice but you can send clients variable because we need it in front.